test/support/commands.js

```diff
@@ -1,5 +1,5 @@
-Cypress.Commands.add('getByTestId', (testId) => cy.get(`[data-testid="${testId}"]`));
-Cypress.Commands.add('getByTitle', (title) => cy.get(`[title="${title}"]`));
+Cypress.Commands.add('getByTestId', (testId, { timeout } = {}) => cy.get(`[data-testid="${testId}"]`, { timeout }));
```

Comment:
Can probably just name the second param `options` and make it a pass-through to avoid the repetition of keys.